modules/elasticsearch_post/consul.tf

```diff
@@ -1,10 +1,27 @@
 resource "consul_node" "es" {
   name    = var.es_consul_service
   address = var.es_endpoint
+
+  meta = {
+    "external-node"  = "true"
+    "external-probe" = "true"
+  }
 }
 
 resource "consul_service" "es" {
   node = consul_node.es.name
   name = var.es_consul_service
   port = var.es_default_access["port"]
+  tags = ["elasticsearch"]
+
+  check {
+    check_id                          = "service:elasticsearch"
+    name                              = "Elasticsearch cluster health check"
+    status                            = "passing"
+    http                              = "${var.es_endpoint}/_cluster/health"
+    method                            = "GET"
+    interval                          = "30s"
+    timeout                           = "10s"
+    deregister_critical_service_after = "600s"
```

Comment:
When the Elasticsearch Consul service gets deregistered, how would we be alerted of the incident?

Reply:
It should work similarly to other registered services. It will appear in Consul web UI as failing red cross under health check (as compared to the '0' gray looking dash symbol).

It will be up to one to funnel out the failing health checks into other mechanisms, if one prefers the alert to appear more aggressively.